snuba/admin/static/clickhouse_queries/query_display.tsx

```diff
@@ -188,30 +207,60 @@ function QueryDisplay(props: {
             size="xl"
           />
         </div>
+        <div style={switchStyle}>
+          <Switch
+            checked={query.clusterless}
+            onChange={(evt: React.ChangeEvent<HTMLInputElement>) =>
+              setClusterlessHost(evt.currentTarget.checked)
+            }
+            onLabel="Input host nodes"
+            offLabel="Auto populate nodes"
```

Comment:
Feel like these labels aren't the best... open to suggestions 👀

Reply:
`onLabel="Manual Host Entry"` make more sense?